packages/nuxt-locokit/src/styles/primevue/_inputswitch.scss

```diff
@@ -4,7 +4,7 @@
 }
 
 .p-inputswitch .p-inputswitch-slider {
-  background: #3f4b5b;
+  background: rgba(63, 75, 91, 0.64); // Custom
```

Comment:
Same here, prefer using CSS vars.

Reply:
cf https://github.com/locokit/locokit/pull/187#discussion_r1116895340

Reply:
same here for @leboiss , don't forget :-)

Reply:
https://github.com/locokit/locokit/issues/211